test/helpers/test.js

```diff
@@ -176,10 +176,48 @@
 
     for ( var i in testLog ) {
         if ( testLog.hasOwnProperty( i ) && testLog[i].push ) {
+
+            //Tile.tiledImage crates circular reference
+            var tileCircularStructureReplacer = function (key, value) {
+                if (value instanceof OpenSeadragon.Tile) {
+                    return {
```

Comment:
This feels brittle; you'll have to keep it up to date over time. A better approach would be to make a copy of the tile (e.g. with `Object.assign`) and remove the `tiledImage` from the copy.